tuxemon/core/monster.py

```diff
@@ -455,7 +457,10 @@ def get_sprite(self, sprite, **kwargs):
         elif sprite == "back":
             surface = tools.load_sprite(self.back_battle_sprite, **kwargs)
         elif sprite == "menu":
-            surface = tools.load_sprite(self.menu_sprite, **kwargs)
+            surface = tools.load_animated_sprite([
+                self.menu_sprite_1, 
+                self.menu_sprite_2],
```

Comment:
what happens if a monster JSON does not implement the 2nd sprite?

Reply:
Not aware of any cases: I assumed that's considered a base requirement for any working monster. At worst the 2nd sprite can be copied and renamed.

I don't have a guard against that in place. If you believe one is necessary I can do so.

Reply:
Please test this with a monster that doesn't implement the 2nd sprite.  If it works, then lets use that.  If it doesn't work, please make sure that animated avatars is optional because I don't want to make work for the artists for this feature.

Reply:
If the second menu sprite is missing, it will instead animate between the first sprite and the "?" sprite (missing monster image).

Reply:
Fixed. If a file isn't found it's simply not added to the animation any more: Both the first and second menu sprites can be missing, the other will just show as a static image instead.